config-model/src/test/java/com/yahoo/vespa/model/container/xml/ContainerModelBuilderTest.java

```diff
@@ -717,6 +717,26 @@ public void secret_store_can_be_set_up() {
         assertEquals("env1", secretStore.getGroups().get(0).environment);
     }
 
+    @Test
+    public void honours_environment_vars() {
+        Element clusterElem = DomBuilderTest.parse(
+                "<jdisc version='1.0'>",
```

Comment:
'jdisc' is soon to be deprecated, so use 'container' instead. (I know a lot of tests still use 'jdisc', but it will be fixed.) Verify that the test still works - you probably need to update the configId in the `getConfig` call below.

Reply:
Fixed